stock_mts_mto_rule/model/procurement.py

```diff
@@ -19,12 +19,21 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ###############################################################################
-from openerp import models, api
+from openerp import models, api, fields
 
 
 class ProcurementOrder(models.Model):
     _inherit = 'procurement.order'
 
+    mts_mto_procurement_id = fields.Many2one(
+        'procurement.order',
+        string="Procurement",
+        copy=False)
+    procurement_ids = fields.One2many(
```

Comment:
Sorry for not mentioning, but this implies to change this one also because it follow the same logic: mts_mto_procurement_ids